hub/core/tensor.py

```diff
@@ -827,7 +827,7 @@ def _sample_shape_provider(self, sample_shape_tensor) -> Callable:
         if self.is_sequence:
 
             def get_sample_shape(global_sample_index: int):
-                shapes = sample_shape_tensor.numpy(
+                shapes = sample_shape_tensor[
```

Comment:
kinda unrelated to the PR, but if you could simplify this formatting then it would be much easier to read.